index.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const app = express()
+const bodyParser = require('body-parser')
+const { getMovieByTitle, getAllMovies, saveNewMovie } = require('./controllers/movie')
+
+
+app.get('/movies', getAllMovies)
+
+
+app.get('/movies/:title', getMovieByTitle)
+
+app.post('/', bodyParser.json(), saveNewMovie)
+
+app.all('*', (req, res) => {
+  return res.send('wrong')
```

Comment:
_Nit_ not sure why you're doing this but its not necessary and shouldn't be left in the code.